Source/GitReleaseManager/Configuration/ConfigurationProvider.cs

```diff
@@ -42,6 +42,8 @@ public static Config Provide(string gitDirectory, IFileSystem fileSystem)
                 }
             }
 
+            _logger.Warning("Yaml not found, that's ok! Learn more at https://gittools.github.io/GitReleaseManager/docs/configuration/default-configuration");
```

Comment:
```suggestion
            _logger.Warning("Yaml not found, that's ok! Learn more at {Url}", "https://gittools.github.io/GitReleaseManager/docs/configuration/default-configuration");
```

I suggest the above change. That way the url will be colorized, and stand out a little when it gets logged.